3rdParty/iresearch/core/formats/columnstore2.cpp

```diff
@@ -368,6 +360,68 @@ class column_base : public column_reader,
     return *stream_;
   }
 
+ protected:
+  column_header& mutable_header() { return hdr_; }
+  void reset_stream(const index_input* stream) { stream_ = stream; }
+
+  bool allocate_buffered_memory(size_t size) {
+    assert(size < static_cast<size_t>(std::numeric_limits<int64_t>::max()));
+    assert(buffered_memory_accounter_);
+    if (!buffered_memory_accounter_(static_cast<int64_t>(size))) {
+      // use string to have always null-terminated name
+      std::string col_name{"<anonymous>"};
+      if (!name().null()) {
+        col_name = name();
+      }
+      IR_FRMT_WARN(
+          "Failed to allocate memory for buffered column id %d name: %s of "
```

Comment:
`%d` is used here for `header().id`, which is of type `irs::field_id`, i.e. `uint64_t`. Should this be `%llu` instead?

Reply:
Good catch 👍  Will fix.